packages/server/lambdas/alfresco/delete-folder.ts

```diff
@@ -17,12 +18,9 @@ let fileEndpointsCache: Array<CategoryDataBase> = [];
 const deleteFolder = async (options: RequestInit, nodeId: string) => {
   const alfrescoCoreAPIUrl = `${getAlfrescoUrlBase()}/alfresco/versions/1`;
   const url = `${alfrescoCoreAPIUrl}/nodes/${nodeId}`;
-  try {
-    const res = await fetch(url, options);
-    return res;
-  } catch (err) {
-    log.error('error:' + err);
-  }
+  const res = await fetch(url, options);
+  const result = (await res.json()) as AlfrescoResponse;
```

Comment:
This returned res instead of result, unlike elsewhere. Any idea if this was on purpose? But this definitely needs to be tested.

Reply:
Seems like this function is not currently being used to delete folders, we use `delete-file` instead. This is because alfresco API works that way, it doesn't care if its a file or a folder, same API for both. Maybe we should check in some other ticket how many of these unused functions we have and decide whether to use them for clarity?

Reply:
In that case, maybe add a comment that about this current situation 👍